roles/lib_openshift/src/class/oc_sdnvalidator.py

```diff
@@ -0,0 +1,58 @@
+# pylint: skip-file
+# flake8: noqa
+
+# pylint: disable=too-many-instance-attributes
+class OCSDNValidator(OpenShiftCLI):
+    ''' Class to wrap the oc command line tools '''
+
+    def __init__(self, kubeconfig):
+        ''' Constructor for OCSDNValidator '''
+        # namespace has no meaning for SDN validation, hardcode to 'default'
+        super(OCSDNValidator, self).__init__('default', kubeconfig)
+
+    def get(self, kind, invalid_filter):
+        ''' return SDN information '''
+
+        rval = self._get(kind)
+        if rval['returncode'] != 0:
+            return False, rval, []
+
+        return True, rval, filter(invalid_filter, rval['results'][0]['items'])
+
+    # pylint: disable=too-many-return-statements
+    @staticmethod
+    def run_ansible(params):
+        ''' run the idempotent ansible code
+
+            params comes from the ansible portion of this module
+        '''
```

Comment:
The indentation of docstrings seem quite inconsistent, I am surprised our linting tools don't bother.
https://www.python.org/dev/peps/pep-0257/